src/search-manager/data/api.ts

```diff
@@ -101,13 +112,13 @@ export interface ContentHit {
    * - After that is the name and usage key of any parent Section/Subsection/Unit/etc.
    */
   breadcrumbs: [{ displayName: string }, ...Array<{ displayName: string, usageKey: string }>];
-  tags: Record<'taxonomy' | 'level0' | 'level1' | 'level2' | 'level3', string[]>;
```

Comment:
We don't necessarily have the `level1`, `level2` and `level3` fields in a `ContentHit`